win32ss/user/user32/user32.spec

```
@@ -1,5 +1,6 @@
 ; Functions exported by Win 2K3 SP2
 @ stdcall ActivateKeyboardLayout(long long) NtUserActivateKeyboardLayout
+@ stdcall -version=0x600+ AddClipboardFormatListener(long)
```

```suggestion
@ stdcall -version=0x600+ AddClipboardFormatListener(ptr)
```
because the parameter is a handle.